app/code/Magento/Sitemap/Controller/Adminhtml/Sitemap/Save.php

```diff
@@ -6,11 +6,70 @@
 namespace Magento\Sitemap\Controller\Adminhtml\Sitemap;
 
 use Magento\Backend\App\Action;
+use Magento\Framework\App\Action\HttpPostActionInterface;
 use Magento\Framework\App\Filesystem\DirectoryList;
 use Magento\Framework\Controller;
 
-class Save extends \Magento\Sitemap\Controller\Adminhtml\Sitemap
+/**
+ * Save XML sitemap action.
+ */
+class Save extends \Magento\Sitemap\Controller\Adminhtml\Sitemap implements HttpPostActionInterface
 {
+    /**
+     * Maximum length of sitemap filename
+     */
+    const MAX_FILENAME_LENGTH = 32;
+
+    /**
+     * @var $_stringValidator
```

Comment:
Please correct propeties' types in phpdocs

Reply:
okay I will review these changes